include/engine/phantom_node.hpp

```diff
@@ -78,7 +76,7 @@ struct PhantomNode
           reverse_segment_id{SPECIAL_SEGMENTID, false},
           name_id(std::numeric_limits<unsigned>::max()), forward_weight(INVALID_EDGE_WEIGHT),
           reverse_weight(INVALID_EDGE_WEIGHT), forward_offset(0), reverse_offset(0),
-          forward_packed_geometry_id(SPECIAL_EDGEID), reverse_packed_geometry_id(SPECIAL_EDGEID),
+          packed_geometry_id(SPECIAL_EDGEID),
```

Comment:
I'm not sure whether we should use GeometryID in this situation or a raw NodeID. For the `m_via_node_geometry_ids` array in the datasources, we needed to pass through the direction from which to read the via-geometry for an edge-based-edge. I'm still trying to determine whether the same thing needs to happen for phantom nodes.